### PR TITLE
add datadog-integrations-core

### PR DESCRIPTION
**Pull request by joshrwolf**
the integrations components needed at runtime by `datadog-agent`

### CODE REVIEW

File: datadog-integrations-core.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  - runs: |
+      python3 -m pip install "./datadog_checks_base[deps, http]" --prefix=/usr --root=${{targets.destdir}}
+
+  - uses: strip
+
+update:
+  enabled: true
+  github:
+    identifier: DataDog/integrations-core
+    use-tag: true
```
Comment (stormqueen1990) [inline datadog-integrations-core.yaml:38]: https://github.com/datadog/integrations-core uses releases. Should we omit `use-tag` so it pulls from the release instead?
```suggestion
```

Comment (joshrwolf) [reply datadog-integrations-core.yaml:38]: from what I can tell, the `ddev` releases are unrelated, what we actually want are the `x.y.z` tags, which map to the upstream agent version

**Review by smoser — CHANGES_REQUESTED** [review]
```
$ tar tvf packages/x86_64/datadog-integrations-core-7.53.0-r0.apk
drwxrwxr-x root/root         0 1969-12-31 19:00 usr/bin/__pycache__
-rw-rw-r-- root/root      3204 1969-12-31 19:00 usr/bin/__pycache__/jp.cpython-312.pyc
-rwxrwxr-x root/root       227 1969-12-31 19:00 usr/bin/ddtrace-run
-rwxrwxr-x root/root      1695 1969-12-31 19:00 usr/bin/jp.py
-rwxrwxr-x root/root       213 1969-12-31 19:00 usr/bin/jsonschema
-rwxrwxr-x root/root       233 1969-12-31 19:00 usr/bin/normalizer
-rwxrwxr-x root/root       214 1969-12-31 19:00 usr/bin/pyspnego-parse
```

There are differing levels of yuck there. The biggest is __pycache__ dir in bin/.

Where we use the `pip install` approach, we really need to also use a virtual env. 
You can see az.yaml for an example of what we're doing there.

**Comment by joshrwolf** [comment]
closing this since I now realize this needs to be part of the main `datadog-agent` package (for datadog reasons)

**Pull request closed without merging**